doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -145,6 +145,9 @@ Solvers for logic and equality
 
       Prints the default tactic used by :tacn:`firstorder` when no rule applies.
 
+   .. cmd:: Set Firstorder Solver @ltac_expr
```

Comment:
I'm not sure if the command adds something that the standard `Set <option>` does not.  If it doesn't, we should remove the command and otherwise hide it in the doc.  WDYT?

```
VERNAC COMMAND EXTEND Firstorder_Set_Solver CLASSIFIED AS SIDEFF
| #[ locality = Tactic_option.tac_option_locality; ] [ "Set" "Firstorder" "Solver" tactic(t) ] -> {
      set_default_solver
        locality
        (Tacintern.glob_tactic t)
  }
END
```

Reply:
The command *is* the `Set <option>` AFAIU. The reason why it's defined as a command instead of using the standard option mechanism is because this option takes a tactic as an argument (as documented) instead of taking a natural or a string. AFAICT, this is the only such non-standard "option". If you think that's confusing, we could also remove the `.. opt:: ` documentation in favor of the `.. cmd::`. The main drawback would be that this "option" would move from the option index (where users might expect to find it given its syntax) to the command index.